tests/unit/drivers/test_chunk2doc_rank_drivers.py

```diff
@@ -100,3 +131,16 @@ def test_chunk2doc_ranker_driver_MinRanker(self):
         for match in doc.matches:
             # match score is computed w.r.t to doc.id
             self.assertEqual(match.score.ref_id, doc.id)
+
+    def test_chunk2doc_ranker_driver_traverse_apply(self):
+        docs = [create_chunk_matches_to_score() for _ in range(1)]
```

Comment:
doc = `create_chunk_matches_to_score()` since it is only one?

Reply:
fixed.